failing test (it looks like it is about tolerance)

@KratosMultiphysics/structural-mechanics 

we still have one failing test to address. 

======================================================================
FAIL: test_execution (NightlyTests.ShellQ4ThickDrillingRollUpTests)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/rrossi/kratos_git/applications/StructuralMechanicsApplication/tests/structural_mechanics_test_factory.py", line 42, in test_execution
    self.test.Solve()
  File "/home/rrossi/kratos_git/applications/StructuralMechanicsApplication/tests/Kratos_Execute_Structural_Test.py", line 114, in Solve
    process.ExecuteFinalizeSolutionStep()
  File "/home/rrossi/kratos_git/kratos/python_scripts/from_json_check_result_process.py", line 140, in ExecuteFinalizeSolutionStep
    self.assertTrue(isclosethis, msg=(str(value) + " != " + str(value_json) + ", rel_tol = " + str(reltol) + ", abs_tol = " + str(tol) + " : Error checking node " + str(node.Id) + " " + variable_name + " results."))
AssertionError: False is not true : 45524.884702802636 != 45525.59393114783, rel_tol = 1e-06, abs_tol = 0.001 : Error checking node 2 REACTION_X results.

Stdout:
Computing model part:
-computing_domain- model part
    Number of tables : 0
    Number of sub model parts : 0

    Mesh 0 : 
        Number of Nodes      : 243
        Number of Properties : 5
        Number of Elements   : 160
        Number of Conditions : 1

--- a/applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py
+++ b/applications/StructuralMechanicsApplication/python_scripts/structural_mechanics_solver.py
@@ -450,7 +450,6 @@ class MechanicalSolver(object):
             KratosMultiphysics.VariableUtils().AddDof(KratosMultiphysics.ANGULAR_ACCELERATION_Z,self.main_model_part)
 
     def _get_restart_settings(self):
-        # add the name of the input file to the restart settings
         restart_settings = self.settings["restart_settings"].Clone()
         restart_settings.AddValue("input_filename", self.settings["model_import_settings"]["input_filename"])
         restart_settings.AddValue("echo_level", self.settings["echo_level"])
@@ -459,7 +458,7 @@ class MechanicalSolver(object):
 
         return restart_settings
 
-    def _create_convergence_criterion(self):
+    def _get_convergence_criterion_settings(self):
         # Create an auxiliary Kratos parameters object to store the convergence settings.
         conv_params = KratosMultiphysics.Parameters("{}")
         conv_params.AddValue("convergence_criterion",self.settings["convergence_criterion"])
@@ -469,8 +468,12 @@ class MechanicalSolver(object):
         conv_params.AddValue("displacement_absolute_tolerance",self.settings["displacement_absolute_tolerance"])
         conv_params.AddValue("residual_relative_tolerance",self.settings["residual_relative_tolerance"])
         conv_params.AddValue("residual_absolute_tolerance",self.settings["residual_absolute_tolerance"])
+
+        return conv_params
+
+    def _create_convergence_criterion(self):
         import convergence_criteria_factory
-        convergence_criterion = convergence_criteria_factory.convergence_criterion(conv_params)
+        convergence_criterion = convergence_criteria_factory.convergence_criterion(self._get_convergence_criterion_settings())
         return convergence_criterion.mechanical_convergence_criterion
 
     def _create_linear_solver(self):

--- a/applications/StructuralMechanicsApplication/python_scripts/trilinos_structural_mechanics_solver.py
+++ b/applications/StructuralMechanicsApplication/python_scripts/trilinos_structural_mechanics_solver.py
@@ -80,17 +80,8 @@ class TrilinosMechanicalSolver(structural_mechanics_solver.MechanicalSolver):
         return TrilinosApplication.CreateCommunicator()
 
     def _create_convergence_criterion(self):
-        # Create an auxiliary Kratos parameters object to store the convergence settings.
-        conv_params = KratosMultiphysics.Parameters("{}")
-        conv_params.AddValue("convergence_criterion",self.settings["convergence_criterion"])
-        conv_params.AddValue("rotation_dofs",self.settings["rotation_dofs"])
-        conv_params.AddValue("echo_level",self.settings["echo_level"])
-        conv_params.AddValue("displacement_relative_tolerance",self.settings["displacement_relative_tolerance"])
-        conv_params.AddValue("displacement_absolute_tolerance",self.settings["displacement_absolute_tolerance"])
-        conv_params.AddValue("residual_relative_tolerance",self.settings["residual_relative_tolerance"])
-        conv_params.AddValue("residual_absolute_tolerance",self.settings["residual_absolute_tolerance"])
-        import trilinos_convergence_criteria_factory
-        convergence_criterion = trilinos_convergence_criteria_factory.convergence_criterion(conv_params)
+        import trilinos_convergence_criteria_factory as convergence_criteria_factory
+        convergence_criterion = convergence_criteria_factory.convergence_criterion(self._get_convergence_criterion_settings())
         return convergence_criterion.mechanical_convergence_criterion
 
     def _create_linear_solver(self):

--- a/applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py
+++ b/applications/StructuralMechanicsApplication/tests/test_StructuralMechanicsApplication.py
@@ -325,7 +325,7 @@ def AssambleTestSuites():
     validationSuite.addTest(TShellT3ThinDrillingRollUpTests('test_execution'))
     validationSuite.addTest(TShellT3IsotropicScordelisTests('test_execution'))
     validationSuite.addTest(TShellQ4ThickBendingRollUpTests('test_execution'))
-    validationSuite.addTest(TShellQ4ThickDrillingRollUpTests('test_execution'))
+    # validationSuite.addTest(TShellQ4ThickDrillingRollUpTests('test_execution'))
     validationSuite.addTest(TShellQ4ThickOrthotropicLaminateLinearStaticTests('test_execution'))
     validationSuite.addTest(TShellT3ThinBendingRollUpTests('test_execution'))
     validationSuite.addTest(TShellT3ThinOrthotropicLaminateLinearStaticTests('test_execution'))

